lib/views/login_signup.dart

```diff
@@ -0,0 +1,539 @@
+                                ),
+                                child: Form(
+                                  key: _formKey2,
+                                  child: Column(
+                                    children: [
+                                      Row(
+                                        mainAxisAlignment: MainAxisAlignment.start,
+                                        children: [
+                                          IconButton(onPressed: (){
+                                            FocusScope.of(context).requestFocus(fnode1);
+                                            change_size();
+                                            }, icon: Icon(Icons.arrow_back_ios),color: Colors.white,),
+                                        ],
+                                      ),
+                                      Padding(
+                                        padding: const EdgeInsets.all(15.0),
+                                        child: Text("WELCOME!",style: TextStyle(color: Colors.white,fontSize: 35),),
+                                      ),
+                                      Row(
+                                        children: [
+                                          Text("Username",style: TextStyle(color: Colors.white,fontSize: 15),),
+                                        ],
+                                      ),
+                                      TextFormField(
+                                        controller:_username,
+                                        focusNode: uname,
+                                        keyboardType: TextInputType.name,
+                                        autovalidateMode: AutovalidateMode.onUserInteraction,
+                                        onFieldSubmitted: (value){
+                                          FocusScope.of(context).requestFocus(password);
+                                        },
+                                        validator: (value){
+                                          if(value == null || value.isEmpty){
+                                            return "Please enter username";
+                                          }
+                                          return null;
+                                        },
+                                        decoration: InputDecoration(
+                                          hintText: "Enter user name",
+                                          enabledBorder: UnderlineInputBorder(
+                                            borderSide: BorderSide(color: Colors.white),
+                                          ),
+                                          hintStyle: TextStyle(color: Colors.grey[400]),
+                                        ),
+                                        style: TextStyle(color: Colors.white),
+                                      ),
+                                      const SizedBox(height: 50,),
```

Comment:
Please do not use the coma at the last parameter when there are two or less than two parameters.